recipes/dremio-arrow/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "dremio-arrow" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dremio-arrow-{{ version }}.tar.gz
+  sha256: 0489ffd1dd0bfb0385a705f5a3d9af3d086e04f0d48702c929414c143e9b160d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7,<3.11
+    - pip
+    - poetry >=1.1.15
+  run:
+    - python >=3.7,<3.11
+    - pandas >=1.3,<=1.4
+    - pyarrow >=7.0,<=9.0
+    - mkdocs-material-extensions >=1.0.3,<2.0.0
```

Comment:
Of all the optional dependencies, why only list this one?

Reply:
Fixed as per reviewer's comments.